weio/bladed_out_file.py

```diff
@@ -0,0 +1,288 @@
+# -*- coding: utf-8 -*-
+import os
+import numpy as np
+import re
+import pandas as pd
+
+class Read_bladed_file:
+    """
+    Read a Bladed out put file (current version is only binary files)
+    
+    Main methods:
+        read: it finds all % and $ files based on selected .$PJ file and calls "DataValue" to read data from all those files
+        DataValue: it calls "SensorLib" method to find information about each %, $ file, and then read data based on those info
+        OrgData: is is called by "DataValue" to sort channels and values
+        toDataFrame: create Pandas dataframe output
+        
+        
+
+    Example: 
+        filename = r'h:\004_Loads\Sim\Bladed\003\Ramp_up\Ramp_up.$PJ'        
+        Output = Read_bladed_file(filename)
+        Output.read()
+        df = Output.toDataFrame()
+        
+    """ 
+    @staticmethod
+    def defaultExtensions():
+        return ['.$PJ']
+
+    @staticmethod
+    def formatName():
+        return 'Bladed output file'
+    
+    
+    
+    def __init__(self,filename):
+        self.file_name = filename
```

Comment:
to be in harmony with the other tools, you would need to trigger a read if a file is given, like this:
```   

 def __init__(self, filename=None, **kwargs):
        self.filename = None
        if filename:
            self.read(filename, **kwargs)
```

Reply:
I have change it. Please check if that's ok.